WORKSPACE

```diff
@@ -1,10 +1,23 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("//build_deps/tf_dependency:tf_configure.bzl", "tf_configure")
+load("//build_deps/toolchains/gpu:cuda_configure.bzl", "cuda_configure")
```

Comment:
Do we need to duplicate this?
 https://github.com/tensorflow/tensorflow/blob/v2.12.0/tensorflow/workspace1.bzl#L7